src/Bicep.Core.UnitTests/FileSystem/FileResolverTests.cs

```diff
@@ -61,5 +62,74 @@ public void TryRead_should_return_expected_results()
             fileContents.Should().BeNull();
             failureMessage.Should().NotBeNull();
         }
+
+        [TestMethod]
+        [ExpectedException(typeof(IOException), AllowDerivedTypes=true)]
+        public void GetDirectories_should_return_expected_results()
+        {
+            var fileResolver = new FileResolver();
+            var tempDir = Path.Combine(Path.GetTempPath(), $"BICEP_TESTDIR_{Guid.NewGuid()}");
+            var tempFile = Path.Combine(tempDir, $"BICEP_TEST_{Guid.NewGuid()}");
+            var tempChildDir = Path.Combine(tempDir, $"BICEP_TESTCHILDDIR_{Guid.NewGuid()}");
+
+            // make parent dir
+            Directory.CreateDirectory(tempDir);
+            fileResolver.GetDirectories(PathHelper.FilePathToFileUrl(tempDir)).Count().Should().Be(0);
```

Comment:
>Count().Should().Be(0) [](start = 79, length = 22)

FYI for next time - there's a .Should().HaveCount(n) - it might produce a better error message, but I wouldn't go changing all the code to use it necessarily.